client/cody-ui/src/chat/TranscriptItem.tsx

```diff
@@ -62,39 +70,51 @@ export const TranscriptItem: React.FunctionComponent<
     feedbackButtonsOnSubmit,
     showFeedbackButtons,
     copyButtonOnSubmit,
+    submitButtonComponent: SubmitButton,
+    chatInputClassName,
 }) => {
     const [formInput, setFormInput] = useState<string>(message.displayText ?? '')
     const textarea =
-        TextArea && beingEdited && editButtonOnSubmit ? (
-            <TextArea
-                className={classNames(styles.chatInput)}
-                rows={5}
-                value={formInput}
-                autoFocus={true}
-                required={true}
-                onInput={({ target }) => {
-                    const { value } = target as HTMLInputElement
-                    setFormInput(value)
-                }}
-                onKeyDown={event => {
-                    if (event.key === 'Escape') {
-                        setBeingEdited(false)
-                    }
+        TextArea && beingEdited && editButtonOnSubmit && SubmitButton ? (
+            <div className={styles.textAreaContainer}>
+                <TextArea
+                    className={classNames(styles.chatInput, chatInputClassName)}
+                    rows={5}
+                    value={formInput}
+                    autoFocus={true}
+                    required={true}
+                    onInput={({ target }) => {
+                        const { value } = target as HTMLInputElement
+                        setFormInput(value)
+                    }}
+                    onKeyDown={event => {
+                        if (event.key === 'Escape') {
+                            setBeingEdited(false)
+                        }
 
-                    if (
-                        event.key === 'Enter' &&
-                        !event.shiftKey &&
-                        !event.nativeEvent.isComposing &&
-                        formInput &&
-                        formInput.trim()
-                    ) {
-                        event.preventDefault()
-                        event.stopPropagation()
+                        if (
+                            event.key === 'Enter' &&
+                            !event.shiftKey &&
+                            !event.nativeEvent.isComposing &&
+                            formInput &&
+                            formInput.trim()
+                        ) {
+                            event.preventDefault()
+                            event.stopPropagation()
```

Comment:
```suggestion
                            event.stopPropagation()
```
I know you didn't add this but I don't think we need it anymore haha 🙇‍♀️